README.md

```diff
@@ -34,3 +34,18 @@ To get involved, please see [Code of Conduct](./CODE_OF_CONDUCT.md) and [contrib
 This repository forms part of the Eiffel Community. Eiffel is a protocol for technology agnostic machine-to-machine communication in continuous integration and delivery pipelines, aimed at securing scalability, flexibility and traceability. Eiffel is based on the concept of decentralized real time messaging, both to drive the continuous integration and delivery system and to document it.
 
 Visit [Eiffel Community](https://eiffel-community.github.io) to get started and get involved.
+
+# Documentation
+
+1. [**GUI Overview**](./wiki/GUI-Overview.md)
+1. [**Subscription Handling**](./wiki/Subscription-Handling.md)
+    -  [**Add Subscription**](./wiki/Add-Subscription.md)
+1. [**Test Rules**](./wiki/Test-Rules.md)
+    - [**Rule testing mechanism via "Test Rules" GUI interface**](./wiki/Test-Rules.md#Rule-testing-mechanism-via-"Test-Rules"-GUI-interface)
+    - [**Introduction**](./wiki/Test-Rules.md#Introduction)
+    - [**Aggregated object**](./wiki/Test-Rules.md#Aggregated-object)
+    - [**Rule set up**](./wiki/Test-Rules.md#Rule-set-up)
+    - [**The most common operation you would do**](./wiki/Test-Rules.md#The-most-common-operation-you-would-do)
+1. [**Eiffel Intelligence Information**](./wiki/Eiffel-Intelligence-Information.md)
```

Comment:
Remove this link and the following 2 links, and corresponding wiki pages, they do not contribute anything useful.